.github/workflows/integration_test.yaml

```diff
@@ -131,7 +131,7 @@ jobs:
         run: bash ${{ inputs.pre-run-script }}
       - name: Run integration tests
         run: |
-          echo "CHARM_NAME=$(yq '.name' metadata.yaml)" >> $GITHUB_ENV
+          echo "CHARM_NAME=$(yq '.name' metadata.yaml | echo UNKNOWN)" >> $GITHUB_ENV
```

Comment:
This will output also an error code. Can we check for the existence of the file instead?